cmds/core/xargs/xargs.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2013-2023 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func main() {
+	if err := run(os.Stdin, os.Stdout, os.Stderr, os.Args[1:]); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func run(stdin io.Reader, stdout, stderr io.Writer, args []string) error {
```

Comment:
I was things args could be ...string, not []string, since they are allowed to be empty.

Reply:
yeah test will look better, updated, thx.